azure/scalardl/locals.tf

```diff
@@ -29,11 +29,4 @@ locals {
     start_on_initial_boot = local.database == "cassandra" ? data.terraform_remote_state.cassandra[0].outputs.cassandra_start_on_initial_boot : false
     provision_ids         = local.database == "cassandra" ? join(",", data.terraform_remote_state.cassandra[0].outputs.cassandra_provision_ids) : ""
   }
```

Comment:
`database` removed from `var.scalardl`, so need to update this part.

```
database = lookup(var.scalardl, "database", "cassandra")
```

Reply:
I think `local.database` is still needed to determine whether to include the Cassandra remote state. https://github.com/scalar-labs/scalar-terraform-examples/blob/5f4105f590cd75786986c2c2f2b88282ff2587f0/azure/scalardl/remote.tf#L10
So I'd like to add only the `database` key again to `azure/scalardl/example.tfvars`. Do you have any idea?

```hcl
scalardl = {
  # resource_type                       = "Standard_B2s"
  ...
  # database                            = "cassandra"
  ...
}
```

Reply:
Since the `database` removed from `var.scalardl` in `scalar-terraform` , I think it's a bit sensitive to specify it again here.

Here are some ideas, I think we should discuss it.

1. Use flag variable (eg., `var.use_cassandra`)
2. Remove all `cassandra` part.
3. Check from env file.
e.g.,
```hcl
use_cassandra = replace(file("./scalardl_container.env"), "SCALAR_DB_STORAGE=cassandra", "") != file("./scalardl_container.env")
```

Reply:
As discussed, I will remove the dependencies on the `cassandra` module first before merging this PR